back/boxtribute_server/business_logic/box_transfer/shipment/fields.py

```diff
@@ -1,13 +1,36 @@
+from datetime import datetime
+
 from ariadne import ObjectType
 
 from ....authz import authorize
+from ....models.definitions.base import Base
 from ....models.definitions.shipment import Shipment
 from ....models.definitions.shipment_detail import ShipmentDetail
 
 shipment = ObjectType("Shipment")
 shipment_detail = ObjectType("ShipmentDetail")
 
 
+@shipment.field("labelIdentifier")
+def resolve_shipment_label_identifier(shipment_obj, _):
+    def first_letters_of_base_name(base_id):
```

Comment:
does it make a performance difference if the function is in the resolver and not in a helper file? In the FE there are cases where you not try to do that since the function definition would be applied each time something is re-rendered.

Reply:
Fair point, the function would be defined everytime the resolver is called which can be multiple times per request.